experimental/table/server/master/src/main/java/alluxio/master/catalog/DefaultCatalogMaster.java

```diff
@@ -111,6 +112,22 @@ public Table getTable(String dbName, String tableName) throws IOException {
     return mCatalog.readTable(dbName, tableName, constraint);
   }
 
+  @Override
+  public void transformTable(String dbName, String tableName, String type, String newTableLocation)
+      throws IOException {
+    Table table = mCatalog.getTable(dbName, tableName);
+    AlluxioURI tableLocation = new AlluxioURI(table.getUdbTable().getBaseLocation());
+    AlluxioURI transformedTableLocation = new AlluxioURI(tableLocation.getScheme(),
```

Comment:
does `transformedTableLocation ` always have the same scheme and authority as the original table location? it shouldn't right?

Reply:
currently, transformation will only transform a table from Alluxio and to Alluxio, so the scheme and authority should always be Alluxio.

Reply:
so for now, can we use `newTableLocation` to construct the `transformedTableLocation` URI?